--- mariadb-10.0.11/mysql-test/lib/mtr_cases.pm.orig	2014-05-29 09:38:59.828057468 +0200
+++ mariadb-10.0.11/mysql-test/lib/mtr_cases.pm	2014-05-29 09:39:26.890028880 +0200
@@ -321,7 +321,7 @@ sub parse_disabled {
       chomp;
       next if /^\s*#/ or /^\s*$/;
       mtr_error("Syntax error in $filename line $.")
-        unless /^\s*(?:([-0-9A-Za-z_]+)\.)?([-0-9A-Za-z_]+)\s*:\s*(.*?)\s*$/;
+        unless /^\s*(?:([-0-9A-Za-z_\/]+)\.)?([-0-9A-Za-z_]+)\s*:\s*(.*?)\s*$/;
       mtr_error("Wrong suite name in $filename line $.")
         if defined $1 and defined $suitename and $1 ne $suitename;
       $disabled{($1 || $suitename || '') . ".$2"} = $3;
