diff -Naurp mariadb-10.1.19/support-files/my-huge.cnf.sh mariadb-10.1.19-patched/support-files/my-huge.cnf.sh
--- mariadb-10.1.19/support-files/my-huge.cnf.sh	2016-11-04 12:15:54.000000000 +0100
+++ mariadb-10.1.19-patched/support-files/my-huge.cnf.sh	2016-12-02 10:29:41.111798361 +0100
@@ -25,6 +25,7 @@ socket		= @MYSQL_UNIX_ADDR@
 
 # The MySQL server
 [mysqld]
+log-error       = @LOG_LOCATION@
 port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
 skip-external-locking
diff -Naurp mariadb-10.1.19/support-files/my-innodb-heavy-4G.cnf.sh mariadb-10.1.19-patched/support-files/my-innodb-heavy-4G.cnf.sh
--- mariadb-10.1.19/support-files/my-innodb-heavy-4G.cnf.sh	2016-11-04 12:15:54.000000000 +0100
+++ mariadb-10.1.19-patched/support-files/my-innodb-heavy-4G.cnf.sh	2016-12-02 10:30:07.647805999 +0100
@@ -42,7 +42,7 @@ socket		= @MYSQL_UNIX_ADDR@
 # The MariaDB server
 #
 [mysqld]
-
+log-error	= @LOG_LOCATION@
 # generic configuration options
 port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
diff -Naurp mariadb-10.1.19/support-files/my-large.cnf.sh mariadb-10.1.19-patched/support-files/my-large.cnf.sh
--- mariadb-10.1.19/support-files/my-large.cnf.sh	2016-11-04 12:15:54.000000000 +0100
+++ mariadb-10.1.19-patched/support-files/my-large.cnf.sh	2016-12-02 10:30:14.744806845 +0100
@@ -25,6 +25,7 @@ socket		= @MYSQL_UNIX_ADDR@
 
 # The MariaDB server
 [mysqld]
+log-error	= @LOG_LOCATION@
 port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
 skip-external-locking
diff -Naurp mariadb-10.1.19/support-files/my-medium.cnf.sh mariadb-10.1.19-patched/support-files/my-medium.cnf.sh
--- mariadb-10.1.19/support-files/my-medium.cnf.sh	2016-11-04 12:15:54.000000000 +0100
+++ mariadb-10.1.19-patched/support-files/my-medium.cnf.sh	2016-12-02 10:30:21.726807677 +0100
@@ -26,6 +26,7 @@ socket		= @MYSQL_UNIX_ADDR@
 
 # The MariaDB server
 [mysqld]
+log-error	= @LOG_LOCATION@
 port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
 skip-external-locking
diff -Naurp mariadb-10.1.19/support-files/my-small.cnf.sh mariadb-10.1.19-patched/support-files/my-small.cnf.sh
--- mariadb-10.1.19/support-files/my-small.cnf.sh	2016-11-04 12:15:54.000000000 +0100
+++ mariadb-10.1.19-patched/support-files/my-small.cnf.sh	2016-12-02 10:30:27.988808423 +0100
@@ -24,6 +24,7 @@ socket		= @MYSQL_UNIX_ADDR@
 
 # The MySQL server
 [mysqld]
+log-error	= @LOG_LOCATION@
 port		= @MYSQL_TCP_PORT@
 socket		= @MYSQL_UNIX_ADDR@
 skip-external-locking
