Upstream bug report: https://mariadb.atlassian.net/browse/MDEV-5124

diff -up mariadb-5.5.33a/storage/tokudb/ft-index/cmake_modules/TokuSetupCTest.cmake.cmakehostname mariadb-5.5.33a/storage/tokudb/ft-index/cmake_modules/TokuSetupCTest.cmake
--- mariadb-5.5.33a/storage/tokudb/ft-index/cmake_modules/TokuSetupCTest.cmake.cmakehostname	2013-10-10 16:21:18.904236776 +0200
+++ mariadb-5.5.33a/storage/tokudb/ft-index/cmake_modules/TokuSetupCTest.cmake	2013-10-10 16:21:29.718243555 +0200
@@ -60,7 +60,7 @@ macro(hostname out)
     COMMAND hostname
     OUTPUT_VARIABLE fullhostname
     OUTPUT_STRIP_TRAILING_WHITESPACE)
-  string(REGEX REPLACE "\\.tokutek\\.com$" "" ${out} ${fullhostname})
+  string(REGEX REPLACE "\\.tokutek\\.com$" "" ${out} "${fullhostname}")
 endmacro(hostname)
 
 ## gather machine info
