--- mariadb-10.0.12/scripts/mysql_config.sh.orig	2014-06-17 11:11:54.444874750 +0200
+++ mariadb-10.0.12/scripts/mysql_config.sh	2014-06-17 11:30:48.244454413 +0200
@@ -176,10 +176,9 @@ Options:
                 pkglibdir     [$pkglibdir]
                 plugindir     [$plugindir]
 EOF
-        exit 0
 }
 
-if test $# -le 0; then usage; fi
+if test $# -le 0; then usage; exit 0; fi
 
 while test $# -gt 0; do
         case $1 in
@@ -198,10 +197,10 @@ while test $# -gt 0; do
             pkgincludedir) echo "$pkgincludedir" ;;
             pkglibdir) echo "$pkglibdir" ;;
             plugindir) echo "$plugindir" ;;
-            *) usage ;;
+            *) echo >&2 "Unknown var: $var";  usage >&2; exit 1 ;;
           esac
           ;;
-        *)         usage ;;
+        *)     echo >&2 "Unknown option: $1"; usage >&2; exit 1 ;;
         esac
 
         shift
