Wrong path used for mysqld

rhbz: #1403416

diff -Naurp mariadb-10.2.9/scripts/galera_recovery.sh mariadb-10.2.9/scripts/galera_recovery.sh_patched
--- mariadb-10.2.9/scripts/galera_recovery.sh	2017-09-25 08:33:25.000000000 +0200
+++ mariadb-10.2.9/scripts/galera_recovery.sh_patched	2017-09-28 09:58:54.242618594 +0200
@@ -68,7 +68,7 @@ parse_arguments() {
 
 wsrep_recover_position() {
   # Redirect server's error log to the log file.
-  eval /usr/sbin/mysqld $cmdline_args --user=$user --wsrep_recover \
+  eval @libexecdir@/mysqld $cmdline_args --user=$user --wsrep_recover \
     --disable-log-error 2> "$log_file"
   ret=$?
   if [ $ret -ne 0 ]; then
