--- mariadb-10.1.11.orig/scripts/CMakeLists.txt	2016-01-28 13:12:51.000000000 +0100
+++ mariadb-10.1.11/scripts/CMakeLists.txt	2016-02-03 12:46:26.306109293 +0100
@@ -320,6 +320,34 @@ ELSE()
       COMPONENT ${${file}_COMPONENT}
      )
   ENDFOREACH()
+
+  # files for systemd
+  SET(SYSTEMD_SCRIPTS
+    mysql.tmpfiles.d
+    mysql.service
+    mysql@.service
+    mysql-prepare-db-dir
+    mysql-wait-ready
+    mysql-wait-stop
+    mysql-check-socket
+    mysql-check-upgrade
+    mysql-scripts-common
+    mysql_config_multilib
+    mysql.init
+    my.cnf
+  )
+  FOREACH(file ${SYSTEMD_SCRIPTS})
+    IF(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${file}.sh)
+      CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/${file}.sh
+        ${CMAKE_CURRENT_BINARY_DIR}/${file} ESCAPE_QUOTES @ONLY)
+    ELSEIF(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${file}.in)
+      CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/${file}.in
+        ${CMAKE_CURRENT_BINARY_DIR}/${file} ESCAPE_QUOTES @ONLY)
+    ELSE()
+      MESSAGE(FATAL_ERROR "Can not find ${file}.sh or ${file}.in in "
+        "${CMAKE_CURRENT_SOURCE_DIR}" )
+    ENDIF()
+  ENDFOREACH()
 ENDIF()
 
 # Configure two scripts from one 'in' file.
