diff -up mariadb-5.5.29/include/myisamchk.h.buffer mariadb-5.5.29/include/myisamchk.h
--- mariadb-5.5.29/include/myisamchk.h.buffer	2013-03-13 09:17:02.802160975 +0100
+++ mariadb-5.5.29/include/myisamchk.h	2013-03-13 09:17:37.383161267 +0100
@@ -138,9 +138,9 @@ typedef struct st_handler_check_param
   /* Following is used to check if rows are visible */
   ulonglong max_trid, max_found_trid;
   ulonglong not_visible_rows_found;
+  ulonglong sort_buffer_length;
   ulonglong use_buffers;                        /* Used as param to getopt() */
-  size_t read_buffer_length, write_buffer_length;
-  size_t sort_buffer_length, sort_key_blocks;
+  size_t read_buffer_length, write_buffer_length, sort_key_blocks;
   time_t backup_time;                           /* To sign backup files */
   ulong rec_per_key_part[HA_MAX_KEY_SEG * HA_MAX_POSSIBLE_KEY];
   double new_rec_per_key_part[HA_MAX_KEY_SEG * HA_MAX_POSSIBLE_KEY];
